.github/workflows/pr.yml

```diff
@@ -5,6 +5,9 @@ on:
     - '*'
 concurrency: pr-${{ github.ref }}
 jobs:
+  quality-check:
+    name: quality-check
+    uses: ./.github/workflows/QC.yml
```

Comment:
Better to use lower-case file name when possible because on Linux file name is case-sensitive. Mixed-case file name likely to cause headache on Linux.